docs/literate/src/files/DGMulti_solver_mesh.jl

```diff
@@ -0,0 +1,220 @@
+                              boundary_conditions=boundary_condition_periodic)
+tspan = (0.0, 0.4)
+ode = semidiscretize(semi, tspan)
+
+alive_callback = AliveCallback(alive_interval=10)
+analysis_callback = AnalysisCallback(semi, interval=100, uEltype=real(dg))
+callbacks = CallbackSet(analysis_callback, alive_callback);
+
+sol = solve(ode, RDPK3SpFSAL49(), abstol=1.0e-6, reltol=1.0e-6,
+            callback=callbacks, save_everystep=false);
+#-
+using Plots
+pd = PlotData2D(sol)
+plot(pd)
+
+
+# ## Simulation with triangular elements
+# Also, we can set another element type. We want to use triangles now.
+using Trixi, OrdinaryDiffEq
+equations = CompressibleEulerEquations2D(1.4)
+initial_condition = initial_condition_weak_blast_wave
+
+# We are using Gauss-Lobatto nodes again, so `approximation_type = SBP()`.
+dg = DGMulti(polydeg = 3,
+             element_type = Tri(),
+             approximation_type = SBP(),
+             surface_flux = flux_lax_friedrichs,
+             volume_integral = VolumeIntegralFluxDifferencing(flux_ranocha))
+
+mesh = DGMultiMesh(dg,
+                   cells_per_dimension=(32, 32), # initial_refinement_level = 5
+                   coordinates_min=(-2.0, -2.0),
+                   coordinates_max=( 2.0,  2.0),
+                   periodicity=true)
+
+semi = SemidiscretizationHyperbolic(mesh, equations, initial_condition, dg,
+                                    boundary_conditions=boundary_condition_periodic)
+tspan = (0.0, 0.4)
+ode = semidiscretize(semi, tspan)
+
+alive_callback = AliveCallback(alive_interval=10)
+analysis_callback = AnalysisCallback(semi, interval=100, uEltype=real(dg))
+callbacks = CallbackSet(analysis_callback, alive_callback);
+
+sol = solve(ode, RDPK3SpFSAL49(), abstol=1.0e-6, reltol=1.0e-6,
+            callback=callbacks, save_everystep=false);
+#-
+using Plots
+pd = PlotData2D(sol)
+plot(pd)
+#-
+plot(pd["rho"])
+plot!(getmesh(pd))
+
+
+# ## Triangular meshes on non-Cartesian domains
+# To use triangulate meshes on a non-Cartesian domain, Trixi uses the package [StartUpDG.jl](https://github.com/jlchan/StartUpDG.jl).
+# The following example is based on [`elixir_euler_triangulate_pkg_mesh.jl`](https://github.com/trixi-framework/Trixi.jl/blob/main/examples/dgmulti_2d/elixir_euler_triangulate_pkg_mesh.jl)
+# and uses a pre-defined mesh from StartUpDG.jl.
+using Trixi, OrdinaryDiffEq
+
+# We want to simulate the smooth initial condition [`initial_condition_convergence_test`](@ref)
+# with source terms [`source_terms_convergence_test`](@ref) for the 2D compressible Euler equations.
+equations = CompressibleEulerEquations2D(1.4)
+initial_condition = initial_condition_convergence_test
+source_terms = source_terms_convergence_test
+
+# We create the solver `DGMulti` with triangular elements (`Tri()`) as before.
+dg = DGMulti(polydeg = 3, element_type = Tri(),
+             surface_flux = flux_lax_friedrichs,
+             volume_integral = VolumeIntegralFluxDifferencing(flux_ranocha))
+
```

Comment:
Which approximation type does this choose by default?

Reply:
This chooses the `Polynomial()` approximation type by default

Reply:
Thanks!